app/shared/language/available-languages.ts

```diff
@@ -3,23 +3,12 @@ import { LanguageSchema } from 'shared/types/commonTypes';
 
 type LanguageCode = 'elastic' | 'ISO639_3' | 'ISO639_1';
 
-type LegacyElasticObject = Record<
-  string,
-  { franc: string; elastic: string; ISO639_1: string | null }
->;
-
-type Language = {
-  label: string;
-  value: string;
-};
-
-const undefinedLanguage: LanguageSchema = {
+const OTHER_LANGUAGE_SCHEMA: LanguageSchema = {
```

Comment:
Why the UPPER_SNAKE_CASE naming here? Maybe just adhere to the usual schema naming convention `otherLanguageSchema`?

Reply:
Got it!